app/src/main/java/com/mapbox/maps/testapp/examples/GesturesActivity.kt

```diff
@@ -71,6 +76,14 @@ class GesturesActivity : AppCompatActivity() {
     gestureAlertsAdapter.cancelUpdates()
   }
 
+  override fun onDestroy() {
+    super.onDestroy()
+    gesturesPlugin.removeOnMoveListener(moveListener)
+    gesturesPlugin.removeOnRotateListener(rotateListener)
+    gesturesPlugin.removeOnScaleListener(scaleListener)
+    gesturesPlugin.removeOnShoveListener(shoveListener)
```

Comment:
Nit: It probably doesn't matter but I would change the order just in case `gesturesPlugin` would do something to the activity.
```suggestion
    gesturesPlugin.removeOnMoveListener(moveListener)
    gesturesPlugin.removeOnRotateListener(rotateListener)
    gesturesPlugin.removeOnScaleListener(scaleListener)
    gesturesPlugin.removeOnShoveListener(shoveListener)
    super.onDestroy()
```